.env.example

```
@@ -24,3 +24,5 @@ PAGOPA_API_URL_TEST="https://pagopa-proxy-test"
 SPID_AUTOLOGIN=lussoluca
 SPID_TESTENV_URL=https://spid-testenv2:8088
 IDP_METADATA_URL=https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml
+SHUTDOWN_SIGNALS="SIGINT SIGTERM"
+SHUTDOWN_TIMEOUT=30000
```

are these seconds or milliseconds or days? *always* specify the unit in the var name

```suggestion
SHUTDOWN_TIMEOUT_MILLIS=30000
```